tools/ci_build/github/azure-pipelines/templates/py-packaging-stage.yml

```diff
@@ -56,6 +56,11 @@ parameters:
    - RelWithDebInfo
    - MinSizeRel
 
+- name: Codeql.SkipTaskAutoInjection
```

Comment:
This is a parameter, not a pipeline variable.  Here is how to setup a var: https://learn.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#user-defined-variables

You can also do it in web ui

Reply:
Ah okay, thank you! I changed it through the web UI